monero-outreach-docs/translations/zh-hans/giant/notes.txt

```diff
@@ -0,0 +1 @@
+Simplified Chinese translation. Some edits made for regionalization.
```

Comment:
This file can be removed. You can point out the reson of the edits in the description of the commit

Reply:
Removed.